solutions/usaco-1210.mdx

```diff
@@ -0,0 +1,128 @@
+---
+id: usaco-1210
+source: USACO Gold 2022 February
+title: Cow Camp
+author: Aakash Gokhale
+---
+
+[Official Analysis](http://usaco.org/current/data/sol_prob2_gold_feb22.html)
+
+## Solution
+We can subtract $1$ from $T$ because we will always get the sample test case and add it to our final answer.
+
+Let $E_x$ be the expected value of $x$ submissions. We are trying to find $E_k$. To get the expected value of the final submission, we can find the relation between $E_x$ and $E_{x - 1}$.
+
+
+To do this we can find $E_x$ in terms of $E_{x -1}$ and $p_i$ which is the probability of getting exactly $i$ test cases.
+
+
+The general rule is to resubmit only if the amount of test cases is less the expected value after $x$ submissions and to not resubmit if it is greater than the expected value. The equality case doesn't matter so we can just change less than into less than or equal (or greater than into greater or equal to).
```

Comment:
> The general rule is to resubmit only if the amount of test cases is less the expected value after $x$ submissions and to not resubmit if it is greater than the expected value.

why is this?

> The equality case doesn't matter

what's the equality case, and _why_ doesn't it matter?

Reply:
Revised